### ISSUE TITLE
`trust-dns` has rebranded to `hickory-dns`

### ISSUE THREAD

**Issue by thomaseizinger** [rustsec/advisory-db#1804]
The `trust-dns` org has rebranded and renamed all of their crates to `hickory-dns-xyz`. This makes all the old crates effectively unmaintained from version 0.23 onwards.

Would it be appropriate to issue a notice to make users aware of this?

**Comment by tarcieri** [comment]
Yes, I would suggest filing an unmaintained crate advisory for the old crate names which informs about the new names

**Comment by bluejekyll** [comment]
Please hold off on merging this #1806, I'd like to discuss the reasoning here in https://github.com/hickory-dns/hickory-dns/issues/2069 and come to a consensus. I worry that this could be unnecessarily disruptive depending on peoples audit settings.
